### ISSUE TITLE
Refactor WP-CLI Doc

### ISSUE THREAD

**Issue by rachelwhitton** [pantheon-systems/documentation#2240]
Re: [Using WP-CLI On The Pantheon Platform](https://pantheon.io/docs/wp-cli)

Priority (Low‚ Medium‚ High): Low

## Issue Description:
The article is a bit underwhelming. It's really slim compared to the [Drush](https://pantheon.io/docs/drush) doc (which.. also needs updatin but that's another issue. #1896 to be exact). 

## Suggested Resolution 
Refactor and amp up the value provided by the WP-CLI documentation.

**Comment by tessak22** [comment]
Excited to dig into this!

**Comment by alexfornuto** [comment]
This issue is blocked by #1896. Once that's completed we'll have a better reference for what we want this doc to be.
